create_ceph_cluster_on_centos.adoc

```diff
@@ -0,0 +1,122 @@
+// vim: tw=79
+= Provisioning Ceph Cluster On CentOs
+
+This document describes the mechanism to create ceph cluster on CentOS nodes
+
+
+== Install OS on Each Virtual Machines
```

Comment:
Wouldn't be better to drop "Virtual" here? Installing ceph on virtual machines doesn't make with exception of some testing scenarios.